app/experimenter/visualization/api/v3/views.py

```diff
@@ -36,6 +38,60 @@ class Statistic:
     COUNT = "count"
 
 
+class JetstreamDataPoint(BaseModel):
+    lower: int = None
+    upper: int = None
+    point: int = None
+    metric: str = None
+    branch: str = None
+    statistic: str = None
+    window_index: str = None
+    comparison: str = None
```

Comment:
Oh and also maybe move all these data types out into their own file so this file is just for the actual views?